tests/envs/test_pybullet_blocks.py

```diff
@@ -148,22 +151,6 @@ def test_pybullet_blocks_picking(env):
         state = env.step(action)
     # The block should now be held.
     assert state.get(block, "held") == 1.0
-    # Test the case where the right finger is on the left side of the block,
```

Comment:
was too lazy to deal with this because hardcoding actions in joint space is now annoying, but if you yell at me i will figure it out

Reply:
no problem